src/IntegrationTests/PrivateClientSpec.cs

```diff
@@ -136,9 +137,9 @@ await client.SubscribeAsync (topicFilter, MqttQualityOfService.AtMostOnce)
             });
 
             await barClient.PublishAsync (new MqttApplicationMessage (fooTopic, new byte[255]), MqttQualityOfService.AtMostOnce);
-            await barClient.PublishAsync (new MqttApplicationMessage (fooTopic, new byte[10]), MqttQualityOfService.AtLeastOnce);
-            await barClient.PublishAsync (new MqttApplicationMessage ("other/topic", new byte[500]), MqttQualityOfService.ExactlyOnce);
-            await barClient.PublishAsync (new MqttApplicationMessage (fooTopic, new byte[50]), MqttQualityOfService.ExactlyOnce);
+            await barClient.PublishAsync (new MqttApplicationMessage (fooTopic, new byte[10]), MqttQualityOfService.AtMostOnce);
```

Comment:
why did this need to change? is there a new default that's in place now? If so, shouldn't the tests use that instead of hardcoding a value that will need to change again if the default ever changes?

Reply:
It changed because the QoS doesn't matter in this test, and using higher QoS is unnecessary. It should have been zero from the begginings